src/components/chatbox/message.module.css

```diff
@@ -0,0 +1,94 @@
+.message {
+	display: flex;
+	flex-direction: row;
+	gap: 12px;
+	padding: 20px 16px;
+	background: var(--token-color-palette-neutral-0);
+}
+
+.message_user {
+	border-bottom: 1px solid var(--token-color-palette-neutral-300);
+}
+
+.message_avatar,
+.message_avatar img {
+	width: 40px;
+	height: 40px;
+	border-radius: 6px;
+}
+
+.message_content {
+	padding-top: 10px; /* simulate vertically centering the first line with avatar/icon */
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+}
+
+.message_user_input {
+	color: var(--token-color-foreground-primary);
+	white-space: pre-line; /* visually respect user-inputted line breaks */
+}
+
+.message_gutter {
+	min-width: 40px;
+}
+
+.message_assistant {
+	background: var(--token-color-surface-strong);
+}
+
+.message_icon {
+	--icon-color: var(--token-color-foreground-highlight-on-surface);
+}
+
+.message_markdown {
+	color: var(--token-color-foreground-primary);
+	& pre {
+		overflow-x: auto;
+		white-space: pre-wrap;
+		word-wrap: break-word;
+	}
+}
+
+.message_AssistantMessageFooter {
+	width: 100%;
+	padding: 24px 0 0;
+}
+
+.message_AssistantMessageFooterHidden {
+	display: none;
+}
+
+.message_divider {
+	display: block;
+	width: 100%;
+	border-bottom: 1px solid var(--token-color-palette-neutral-300);
+	margin-bottom: 24px;
+}
+
+.message_actionButtons {
+	display: flex;
+	flex-direction: row;
+	gap: 16px;
+}
+
+.rating_like {
+	border: 1px solid var(--token-color-border-success) !important;
+	background: var(--token-color-surface-success) !important;
+	color: var(--token-color-foreground-success) !important;
+}
+.rating_dislike {
+	border: 1px solid var(--token-color-border-critical) !important;
+	background: var(--token-color-surface-critical) !important;
+	color: var(--token-color-foreground-critical) !important;
```

Comment:
Could you expand on these?

Reply:
@kellyschummer could we simply do a back arrow if you want to change your selection instead of this red/green button pattern?

Reply:
💭 Would it be acceptable to use the "primary" state to indicate selection here, rather than custom coloration? Here's a quick mockup via Swingset for clarity:

<img width="1061" alt="CleanShot 2023-09-21 at 16 02 05@2x" src="https://github.com/hashicorp/dev-portal/assets/4624598/c7e6b7ae-544e-4333-9736-7477dc87a342">

<details>

<summary>Swingset code for Button</summary>

```jsx
const FeedbackExample = () => (
  <>
    <div data-theme="light">
      <div
        style={{
          display: "flex",
          gap: "24px",
          padding: "24px",
          background: "var(--token-color-surface-primary)",
        }}
      >
        <div
          style={{
            display: "grid",
            gap: "8px",
            gridTemplateColumns: "min-content min-content",
          }}
        >
          <span style={{ color: "var(--token-color-foreground-primary)" }}>
            Initial
          </span>
          <span></span>
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="yes"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-up" />}
          />
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="No"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-down" />}
          />
        </div>

        <div
          style={{
            display: "grid",
            gap: "8px",
            gridTemplateColumns: "min-content min-content",
          }}
        >
          <span style={{ color: "var(--token-color-foreground-primary)" }}>
            1
          </span>
          <span></span>
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="Yes"
            icon={<SwingsetTestIcon name="thumbs-up" />}
          />
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="No"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-down" />}
          />
        </div>

        <div
          style={{
            display: "grid",
            gap: "8px",
            gridTemplateColumns: "min-content min-content",
          }}
        >
          <span style={{ color: "var(--token-color-foreground-primary)" }}>
            -1
          </span>
          <span></span>
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="Yes"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-up" />}
          />
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="No"
            icon={<SwingsetTestIcon name="thumbs-down" />}
          />
        </div>
      </div>
    </div>

    <div data-theme="dark">
      <div
        style={{
          display: "flex",
          gap: "24px",
          padding: "24px",
          background: "var(--token-color-surface-primary)",
        }}
      >
        <div
          style={{
            display: "grid",
            gap: "8px",
            gridTemplateColumns: "min-content min-content",
          }}
        >
          <span style={{ color: "var(--token-color-foreground-primary)" }}>
            Initial
          </span>
          <span></span>
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="yes"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-up" />}
          />
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="No"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-down" />}
          />
        </div>

        <div
          style={{
            display: "grid",
            gap: "8px",
            gridTemplateColumns: "min-content min-content",
          }}
        >
          <span style={{ color: "var(--token-color-foreground-primary)" }}>
            1
          </span>
          <span></span>
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="Yes"
            icon={<SwingsetTestIcon name="thumbs-up" />}
          />
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="No"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-down" />}
          />
        </div>

        <div
          style={{
            display: "grid",
            gap: "8px",
            gridTemplateColumns: "min-content min-content",
          }}
        >
          <span style={{ color: "var(--token-color-foreground-primary)" }}>
            -1
          </span>
          <span></span>
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="Yes"
            color="secondary"
            icon={<SwingsetTestIcon name="thumbs-up" />}
          />
          <Button
            onClick={() => console.log("clicked button")}
            aria-label="No"
            icon={<SwingsetTestIcon name="thumbs-down" />}
          />
        </div>
      </div>
    </div>
  </>
);
```


</details>

Reply:
@thiskevinwang @zchsh - I like the above approach with the primary state but would recommend we keep consistent with our active state -  Kevin and I agree and decided the simplify this flow even further and remove the input- Keeping the engagement to the 👍 👎 . 
<img width="418" alt="Screenshot 2023-09-21 at 1 19 14 PM" src="https://github.com/hashicorp/dev-portal/assets/89168672/6630be52-6c97-414f-a8df-acd5ace259ab">

Reply:
FYI @kellyschummer, I'm going to merge this PR, and follow up with the bulk of [VQA](https://app.asana.com/0/1100423001970639/1205523765153638/f) in a separate PR. 
- That'll include deleting the "Feedback Text Input"